### PR TITLE
[KOA-3657] Update nudger component to use the new spacing grid

### PR DESCRIPTION
**Pull request by anambl**
Before: https://backpack.github.io/storybook/?path=/story/bpk-component-nudger--default
After: https://backpack.github.io/storybook-prs/2248/?path=/story/bpk-component-nudger--default

Remember to include the following changes:

- [x] `UNRELEASED.md` (See [How to write a good changelog entry](https://github.com/Skyscanner/backpack/blob/main/CHANGELOG_FORMAT.md))
- [ ] `README.md`
- [ ] Tests
- [ ] Docs (either update [backpack-docs](https://github.com/Skyscanner/backpack-docs) now, or create a follow up ticket)

### CODE REVIEW

File: packages/bpk-component-nudger/src/BpkNudger.module.scss [hunk 1]
```diff
@@ -39,7 +41,7 @@
     }
 
     &--numeric {
-      $size: $bpk-button-line-height + ($bpk-button-padding-y * 2);
+      $size: bpk-spacing-lg() + (bpk-spacing-md() * 2);
```
Comment (olliecurtis) [inline packages/bpk-component-nudger/src/BpkNudger.module.scss:44]: For the `$bpk-button-line-height`, has this changed to aid with the new grid size, as keeping it in use causes the size to be off?

**Review by olliecurtis — CHANGES_REQUESTED** [review]
Just a small comment from me otherwise looks great!

**Review by olliecurtis — APPROVED** [review]
LGTM!

File: packages/bpk-component-nudger/src/BpkNudger.module.scss (hunk 1 above)
Comment (anambl) [reply packages/bpk-component-nudger/src/BpkNudger.module.scss:44]: `$bpk-button-line-height` uses a spacing token under the hood and also it's used for setting the width

**Pull request merged**